website/source/docs/drivers/qemu.html.md

```diff
@@ -47,6 +48,8 @@ The `qemu` driver supports the following configuration in the job spec:
   If the host machine has `qemu` installed with KVM support, users can specify
   `kvm` for the `accelerator`. Default is `tcg`.
 
+* `graceful_shutdown` `(bool: false)` - Using the [qemu monitor](https://en.wikibooks.org/wiki/QEMU/Monitor), send an ACPI shutdown signal to virtual machines rather than simply terminating them. This emulates a physical power button press, and gives instances a chance to shut down cleanly. If the VM is still running after ``kill_timeout``, it will be forcefully terminated. (Note that [prior to qemu 2.10.1](https://github.com/qemu/qemu/commit/ad9579aaa16d5b385922d49edac2c96c79bcfb6), the monitor socket path is limited to 108 characters. Graceful shutdown will be disabled if qemu is < 2.10.1 and the generated monitor path exceeds this length. You may encounter this issue if you set long [data_dir](https://www.nomadproject.io/docs/agent/configuration/index.html#data_dir) or [alloc_dir](https://www.nomadproject.io/docs/agent/configuration/client.html#alloc_dir) paths.)
```

Comment:
Wrap at 80 characters and indent lines with 2 spaces (we're considering loosening this style, but now we're trying to stay consistent).

> If the VM is still running after ``kill_timeout``, it will be forcefully terminated.

Huh, I thought if you wrote the shutdown message to the monitor's socket properly the `Kill()` method returned and Nomad would consider the VM dead even if it wasn't? Perhaps my comment on that code above is wrong?